internal/hcp/registry/metadata.cicd.go

```diff
@@ -0,0 +1,103 @@
+package registry
```

Comment:
I think that the fact that those three files are prefixed with `metadata` could mean that `metadata` could be a package under hcp/registry.